formidable/migrations/0010_auto_20180731_1337.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-07-31 13:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('formidable', '0009_field_parameters'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='field',
+            name='parameters',
+            field=jsonfield.fields.JSONField(
+                blank=True, default={}, null=True
```

Comment:
I think that the best would be to merge this migration with the 0009, and keep its "semantic" name, instead of the automatically generated one.

Reply:
Ok for me, do you want to do it?